rest/admin_api.go

```diff
@@ -112,7 +112,7 @@ func (h *handler) handleCreateDB() error {
 		// load database in-memory for non-persistent nodes
 		if _, err := h.server.AddDatabaseFromConfigFailFast(DatabaseConfig{DbConfig: *config}); err != nil {
 			if errors.Is(err, base.ErrAuthError) {
-				return base.HTTPErrorf(http.StatusForbidden, "auth failure accessing provided bucket using database credentials: %s", base.Metadata(bucket))
+				return base.HTTPErrorf(http.StatusForbidden, "auth failure using provided bucket credentials for database %s", base.Metadata(config.Name))
```

Comment:
This should be base.MD() not Metadata. We don't use Metadata directly and always use MD(). It probably works in this case as its a string but MD() is used to transform variables which may not be strings.

Reply:
Oops, didn't spot that. Thanks